repository/observation_repository.go

```diff
@@ -0,0 +1,82 @@
+package repository
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	"github.com/ONSdigital/dp-dimension-importer/common"
+	"github.com/ONSdigital/dp-dimension-importer/logging"
+	"github.com/ONSdigital/dp-dimension-importer/model"
+	bolt "github.com/johnnadratowski/golang-neo4j-bolt-driver"
+)
+
+const (
+	uniqueObsConstraintStmt = "CREATE CONSTRAINT ON (o:`%s`) ASSERT o.rowIndex IS UNIQUE"
+	observationLabelFmt     = "_%s_observation"
+)
+
+// NewObservationRepository returns a new ObservationRepository. A bolt.Conn will be obtained from the supplied connectionPool.
+// The obtained bolt.Conn will be used for the life time of the InstanceRepository struct
```

Comment:
Should be ObservationRepository in the comment.

Reply:
Fixed 👍